pkg/labels/filter_test.go

```diff
@@ -33,7 +33,7 @@ func (s *LabelsPrefCfgSuite) TestFilterLabels(c *C) {
 	}
 
 	dlpcfg := defaultLabelPrefixCfg()
-	d, err := parseLabelPrefix("!ignor[eE]")
+	d, err := parseLabelPrefix(":!ignor[eE]")
```

Comment:
Is this change necessary?

Reply:
It is, to cover the if branch you said is not covered by the unit tests.

Reply:
I thought you were accidentally changing behavior.

Reply:
Mu bad, I should have called out this change in the commit message.